packages/lib/server/repository/profile.ts

```diff
@@ -249,20 +250,68 @@ export class ProfileRepository {
     });
   }
 
+  static async createManyForExistingUsers({
+    users,
+    organizationId,
+    orgAutoAcceptEmail,
+  }: {
+    users: { id: number; username: string | null; email: string }[];
+    organizationId: number;
+    orgAutoAcceptEmail: string;
+  }) {
+    return await prisma.profile.createMany({
+      data: users.map((user) => ({
+        uid: ProfileRepository.generateProfileUid(),
+        userId: user.id,
+        organizationId,
+        username: user?.username || getOrgUsernameFromEmail(user.email, orgAutoAcceptEmail),
+        movedFromUser: {
+          connect: {
+            id: user.id,
+          },
+        },
+      })),
+      skipDuplicates: true,
+    });
+  }
+
+  static async createManyPromise({
```

Comment:
`createMany` wasn't returning a promise because it was being used in a `$transaction`. So we need to create this method if we're using it outside of the `$transaction`